services/core-web/src/components/mine/MineDashboard.js

```diff
@@ -2,6 +2,7 @@ import React, { Component } from "react";
 import { connect } from "react-redux";
 import PropTypes from "prop-types";
 import { bindActionCreators } from "redux";
+import moment from "moment";
```

Comment:
moment is deprecated. suggest using 'date-fns'

Reply:
Sounds good to me.  I was unsure about introducing a new date library at this point in the project, but I much prefer date-fns.

Reply:
just moment is deprecated and the new code should use actively maintained libraries. Plus moment.js is not immutable.

Reply:
💯 Updating it now